plugins/services/src/js/service-configuration/ServiceNetworkingConfigSection.js

```diff
@@ -79,8 +79,10 @@ class ServiceNetworkingConfigSection extends ServiceConfigBaseSectionDisplay {
                 prop: keys.protocol,
                 className: getColumnClassNameFn(),
                 render(prop, row) {
-                  return getDisplayValue(row[prop])
-                    .split(',').join(', ');
+                  let protocol = row[prop] || '';
+                  protocol = protocol.replace(',', ', ');
```

Comment:
Use `.replace(/,/g, ', ')` because `.replace('string')` will replace only the first occurrence. And while at it, why don't you try `/,.*/g` just to be safe?